#! /usr/bin/env python

PACKAGE='jsk_perception'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add('file_name', str_t, 0, 'image file name to publish as image topic', default='image.png')

exit(gen.generate(PACKAGE, "jsk_perception", "ImagePublisher"))
